### PR TITLE
Fix empty list for image

### PR DESCRIPTION
**Pull request by MV88**
## Description

- [ ] Added a message of empty list of images in media editor.
- [ ] added link to user guide for the three tabs (images, maps, videos)
- [ ] removed default image for new geostory config

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [x] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#5310

**What is the new behavior?**

A message appear if not images are in the media editore image list

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/translations/data.de-DE.json [hunk 1]
```diff
@@ -2243,11 +2243,14 @@
                 "confirmRemoveResource": "Möchten Sie diese Medienressource aus der Story entfernen?",
                 "confirmRemoveUsedResource": "Diese Ressource wird auch aus allen Abschnitten / Inhalten entfernt, in denen sie verwendet wird und sie bleiben leer. Möchten Sie es aus der Geschichte entfernen?"
             },
+            "imageList": {
+                "emptyList": "Klicken Sie auf das Pluszeichen, um ein neues Bild hinzuzufügen. Weitere Informationen finden Sie im Benutzerhandbuch <a href='https://mapstore.readthedocs.io/en/latest/user-guide/media-editor-window/#images' target='_blank'/> hier <a/>"
```
Comment (offtherailz) [inline web/client/translations/data.de-DE.json:2247]: Wrong link tag (here and in other languages)
It have to end with `</a>` not `</a>`

File: web/client/translations/data.it-IT.json [hunk 2]
```diff
@@ -2244,11 +2244,14 @@
                 "confirmRemoveResource": "Vuoi rimuovere questa risorsa multimediale dalla storia?",
                 "confirmRemoveUsedResource": "Questa risorsa sarà rimossa anche da tutte le sezioni/contenuti che rimarrano vuoti. La vuoi rimuovere dalla storia?"
             },
+            "imageList": {
+                "emptyList": "Premere il pulsante più per aggiungere una nuova immagine. Per maggiori informazioni consulta <a href='https://mapstore.readthedocs.io/en/latest/user-guide/media-editor-window/#images' target='_blank'/>questa</a> guida"
+            },
             "mapList": {
-                "emptyList": "Premere il pulsante più per creare una mappa o selezionare un altro servizio di mappe."
+                "emptyList": "Premere il pulsante più per creare una mappa o selezionare un altro servizio di mappe. Per maggiori informazioni consulta <a href='https://mapstore.readthedocs.io/en/latest/user-guide/media-editor-window/#maps' target='_blank'/>questa<a/> guida"
```
Comment (offtherailz) [inline web/client/translations/data.it-IT.json:2251]:
```suggestion
                "emptyList": "Premere il pulsante più per creare una mappa o selezionare un altro servizio di mappe. Per maggiori informazioni consulta <a href='https://mapstore.readthedocs.io/en/latest/user-guide/media-editor-window/#maps' target='_blank'/>questa</a> guida"
```

File: web/client/translations/data.it-IT.json [hunk 3]
```diff
@@ -2244,11 +2244,14 @@
...
             },
             "videoList": {
-                "emptyList": "Premere il pulsante più per aggiungere un nuovo video"
+                "emptyList": "Premere il pulsante più per aggiungere un nuovo video. Per maggiori informazioni consulta <a href='https://mapstore.readthedocs.io/en/latest/user-guide/media-editor-window/#videos' target='_blank'/>questa<a/> guida"
```
Comment (offtherailz) [inline web/client/translations/data.it-IT.json:2254]:
```suggestion
                "emptyList": "Premere il pulsante più per aggiungere un nuovo video. Per maggiori informazioni consulta <a href='https://mapstore.readthedocs.io/en/latest/user-guide/media-editor-window/#videos' target='_blank'/>questa</a> guida"
```

**Comment by offtherailz** [comment]
I fixed it with inline comments. Please  double check translations than we can merge it @MV88

**Comment by MV88** [comment]
> I fixed it with inline comments. Please double check translations than we can merge it @MV88

seems good now

**Comment by offtherailz** [comment]
Please create a backport for 2020.02.xx

**Pull request merged**